cpp/include/cuvs/core/c_api.h

```diff
@@ -17,6 +17,7 @@
 #pragma once
 
 #include <cuda_runtime.h>
+#include <stdbool.h>
```

Comment:
I'm not a C wizard -- but is this include really needed? It sounds like it's a compatibility shim for super old-school C code. https://stackoverflow.com/questions/47374356/why-use-stdbool-h-instead-of-bool

I'd be happy to learn more -- but this might have just been an include automatically added by an IDE?